go.mod

```diff
@@ -239,6 +239,8 @@ require (
 // Updating causes breaking changes.
 require github.com/mailgun/minheap v0.0.0-20170619185613-3dbe6c6bf55f // indirect
 
+require github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc // indirect
+
```

Comment:
move it back to the block below? not sure what happened here.

Reply:
Ah, the `go-spew` move was unintended. I've reverted that now, thanks for noticing.